ext/standard/tests/http/bug73297.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Bug #73297 (Ignore 100 Continue returned by HTTP/1.1 servers)
+--INI--
+allow_url_fopen=1
+--SKIPIF--
+<?php require 'server.inc'; http_server_skipif('tcp://127.0.0.1:12342'); ?>
+--FILE--
+<?php
+require 'server.inc';
+
+$ctx = stream_context_create();
```

Comment:
Ah, well spotted, just not cleaned up from previous version (was also present in my PHPT for PR#1902)